openfe/setup/methods/openmm/equil_rbfe_methods.py

```diff
@@ -427,36 +445,118 @@ def dG(self):
         ----
         * Check this holds up completely for SAMS.
         """
-        dG, _ = self._analyzer.get_free_energy()
-        dG = (dG[0, -1] * self._analyzer.kT).in_units_of(
-            omm_unit.kilocalories_per_mole)
+        raise NotImplementedError()
+
+        dGs = []
+        weights = []
+
+        for analyzer in self._analyzers:
+            dG, _ = analyzer.get_free_energy()
+            dG = (dG[0, -1] * analyzer.kT).in_units_of(
+                omm_unit.kilocalories_per_mole)
 
-        return dG
+            weight = analyzer._get_equilibration_data()[2]
 
-    def dG_error(self):
+            dGs.append(dG)
+            weights.append(weight)
+
+        return np.average(dGs, weights=weights)
+
+    def get_uncertainty(self):
         """The uncertainty/error in the dG value"""
+        # https: // en.wikipedia.org / wiki / Inverse - variance_weighting ?
+        raise NotImplementedError()
         _, error = self._analyzer.get_free_energy()
         error = (error[0, -1] * self._analyzer.kT).in_units_of(
             omm_unit.kilocalories_per_mole)
 
         return error
 
+    def get_rate_of_convergence(self):
+        return 0
+
+
+class RelativeLigandTransform(gufe.Protocol):
+    _results_cls = RelativeLigandTransformResult
+
+    def __init__(self, settings: RelativeLigandTransformSettings):
+        super().__init__(settings)
+
+    @classmethod
+    def _default_settings(cls) -> RelativeLigandTransformSettings:
+        """A dictionary of initial settings for this creating this Protocol
```

Comment:
todo: hack gufe to append this docstring onto abc method

Reply:
Raise issue?